src/common/schema/application.js

```diff
@@ -395,6 +395,9 @@ module.exports = {
     veteranOtherIncome: ['understandsFinancialDisclosure'],
     spouseGrossIncome: ['understandsFinancialDisclosure'],
     spouseNetIncome: ['understandsFinancialDisclosure'],
-    spouseOtherIncome: ['understandsFinancialDisclosure']
+    spouseOtherIncome: ['understandsFinancialDisclosure'],
```

Comment:
Right, I see what you're saying.  I was thinking of it in terms of a user filling out the form, not the final validation.  Will fix in #274!